cli/azd/pkg/tools/python/python.go

```diff
@@ -98,7 +98,7 @@ func (cli *PythonCli) InstallRequirements(ctx context.Context, workingDir, envir
 		installCmd := fmt.Sprintf("%s -m pip install -r %s", pyString, requirementFile)
 		commands := []string{envActivation, installCmd}
 
-		runArgs := exec.NewRunArgs("").WithCwd(workingDir)
+		runArgs := exec.NewRunArgs(pyString).WithCwd(workingDir)
```

Comment:
pyString is set on previous line  (98).  This would result in a double `python python` call.

Either revert this, or remove pyString from `installCmd`

Reply:
It doesn't. This is ignored in the `RunList` function.